translate/convert/po2web2py.py

```diff
@@ -64,7 +64,7 @@ def convertstore(self, inputstore, includefuzzy):
 
 
 def convertpy(inputfile, outputfile, templatefile=None, includefuzzy=False,
-              outputthreshold=None, **kwargs):
+              outputthreshold=None):
```

Comment:
The remove `**kwargs` is an unrelated change. Please put it into a separate commit.